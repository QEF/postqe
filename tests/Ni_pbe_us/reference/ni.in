 &control
    calculation = 'scf'  	 			
    prefix='Ni'      						
    pseudo_dir = './',  				
    outdir='./tmp/'						     
 /
 &system
    ibrav=2, 							
    celldm(1) =6.65,					
    nat=  1, 						
    ntyp= 1,							
    nspin=2,							
    starting_magnetization(1)=0.5,				
    occupations='smearing',					
    smearing='mp',						
    degauss=0.02,						
    ecutwfc =40							
    ecutrho =600.0						
 /
 &electrons
    conv_thr =  1.0d-10     					
    mixing_mode = 'TF'      					
 /
 &ions
 /
 &cell
 / 
ATOMIC_SPECIES
 Ni  58.6934  Ni.pbe-nd-rrkjus.UPF
ATOMIC_POSITIONS {alat}
 Ni 0 0 0
K_POINTS automatic
  3 3 3 1 1 1
