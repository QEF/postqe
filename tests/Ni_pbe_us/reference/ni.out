
     Program PWSCF v.6.1 starts on 17Jul2017 at 11:22:15 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4
     Waiting for input...
     Reading input from standard input
Warning: card  &IONS ignored
Warning: card  / ignored
Warning: card  &CELL ignored
Warning: card  / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file Ni.pbe-nd-rrkjus.UPF: wavefunction(s)  4S renormalized

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         230      63     21                 4549      641     132
     Max         232      64     22                 4550      642     135
     Sum         925     253     85                18197     2565     531



     bravais-lattice index     =            2
     lattice parameter (alat)  =       6.6500  a.u.
     unit-cell volume          =      73.5199 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =        10.00
     number of Kohn-Sham states=            9
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     600.0000  Ry
     convergence threshold     =      1.0E-10
     mixing beta               =       0.7000
     number of iterations used =            8  TF        mixing
     Exchange-correlation      =  SLA  PW   PBE  PBE ( 1  4  3  4 0 0)

     celldm(1)=   6.650000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.500000   0.000000   0.500000 )  
               a(2) = (   0.000000   0.500000   0.500000 )  
               a(3) = (  -0.500000   0.500000   0.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.000000 -1.000000  1.000000 )  
               b(2) = (  1.000000  1.000000  1.000000 )  
               b(3) = ( -1.000000  1.000000 -1.000000 )  


     PseudoPot. # 1 for Ni read from file:
     ./Ni.pbe-nd-rrkjus.UPF
     MD5 check sum: 8081f0a005c9a5470caab1a58e82ecb2
     Pseudo is Ultrasoft + core correction, Zval = 10.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 1203 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2
     Q(r) pseudized with 0 coefficients 


     atomic species   valence    mass     pseudopotential
        Ni            10.00    58.69340     Ni( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Ni           0.500

     48 Sym. Ops., with inversion, found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Ni  tau(   1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=     6  Methfessel-Paxton smearing, width (Ry)=  0.0200
                       cart. coord. in units 2pi/alat
        k(    1) = (  -0.1666667   0.1666667   0.1666667), wk =   0.0740741
        k(    2) = (   0.5000000  -0.5000000   0.8333333), wk =   0.2222222
        k(    3) = (   0.1666667  -0.1666667   0.5000000), wk =   0.2222222
        k(    4) = (  -0.1666667  -1.1666667   0.1666667), wk =   0.2222222
        k(    5) = (  -0.5000000  -0.8333333  -0.1666667), wk =   0.2222222
        k(    6) = (   0.5000000  -0.5000000  -0.5000000), wk =   0.0370370

     Dense  grid:    18197 G-vectors     FFT dimensions: (  40,  40,  40)

     Smooth grid:     2565 G-vectors     FFT dimensions: (  20,  20,  20)

     Estimated max dynamical RAM per process >      20.25 MB

     Estimated total dynamical RAM >      80.99 MB
     Generating pointlists ...
     new r_m :   0.2917 (alat units)  1.9397 (a.u.) for type    1

     Check: negative/imaginary core charge=   -0.000002    0.000000

     Initial potential from superposition of free atoms

     starting charge    9.99954, renormalised to   10.00000
     Starting wfc are    6 randomized atomic wfcs +    3 random wfc

     total cpu time spent up to now is        1.0 secs

     per-process dynamical memory:    12.2 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.6

     total cpu time spent up to now is        1.5 secs

     total energy              =     -85.70920970 Ry
     Harris-Foulkes estimate   =     -85.78217894 Ry
     estimated scf accuracy    <       0.59838186 Ry

     total magnetization       =     1.55 Bohr mag/cell
     absolute magnetization    =     1.60 Bohr mag/cell

     iteration #  2     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  5.98E-03,  avg # of iterations =  1.4

     total cpu time spent up to now is        1.8 secs

     total energy              =     -85.85998488 Ry
     Harris-Foulkes estimate   =     -85.86748604 Ry
     estimated scf accuracy    <       0.13136216 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.07 Bohr mag/cell

     iteration #  3     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.31E-03,  avg # of iterations =  1.0

     total cpu time spent up to now is        2.1 secs

     total energy              =     -85.88889779 Ry
     Harris-Foulkes estimate   =     -85.87759534 Ry
     estimated scf accuracy    <       0.01687285 Ry

     total magnetization       =     0.82 Bohr mag/cell
     absolute magnetization    =     0.99 Bohr mag/cell

     iteration #  4     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.69E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is        2.4 secs

     total energy              =     -85.89496730 Ry
     Harris-Foulkes estimate   =     -85.89484554 Ry
     estimated scf accuracy    <       0.00028874 Ry

     total magnetization       =     0.61 Bohr mag/cell
     absolute magnetization    =     0.81 Bohr mag/cell

     iteration #  5     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.89E-06,  avg # of iterations =  2.0

     total cpu time spent up to now is        2.7 secs

     total energy              =     -85.89506554 Ry
     Harris-Foulkes estimate   =     -85.89507120 Ry
     estimated scf accuracy    <       0.00006938 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.81 Bohr mag/cell

     iteration #  6     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  6.94E-07,  avg # of iterations =  1.5

     total cpu time spent up to now is        3.1 secs

     total energy              =     -85.89509391 Ry
     Harris-Foulkes estimate   =     -85.89510084 Ry
     estimated scf accuracy    <       0.00002011 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration #  7     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.01E-07,  avg # of iterations =  1.3

     total cpu time spent up to now is        3.4 secs

     total energy              =     -85.89509934 Ry
     Harris-Foulkes estimate   =     -85.89509899 Ry
     estimated scf accuracy    <       0.00000262 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration #  8     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.62E-08,  avg # of iterations =  1.0

     total cpu time spent up to now is        3.7 secs

     total energy              =     -85.89509937 Ry
     Harris-Foulkes estimate   =     -85.89509957 Ry
     estimated scf accuracy    <       0.00000045 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration #  9     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.51E-09,  avg # of iterations =  1.5

     total cpu time spent up to now is        4.1 secs

     total energy              =     -85.89509966 Ry
     Harris-Foulkes estimate   =     -85.89509964 Ry
     estimated scf accuracy    <       0.00000015 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration # 10     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.53E-09,  avg # of iterations =  1.2

     total cpu time spent up to now is        4.4 secs

     total energy              =     -85.89509969 Ry
     Harris-Foulkes estimate   =     -85.89509969 Ry
     estimated scf accuracy    <          2.7E-09 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration # 11     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.68E-11,  avg # of iterations =  1.4

     total cpu time spent up to now is        4.7 secs

     total energy              =     -85.89509969 Ry
     Harris-Foulkes estimate   =     -85.89509969 Ry
     estimated scf accuracy    <          1.6E-10 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration # 12     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.55E-12,  avg # of iterations =  1.6

     total cpu time spent up to now is        5.0 secs

     total energy              =     -85.89509969 Ry
     Harris-Foulkes estimate   =     -85.89509969 Ry
     estimated scf accuracy    <          1.3E-10 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     iteration # 13     ecut=    40.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.27E-12,  avg # of iterations =  1.2

     Magnetic moment per site:
     atom:    1    charge:    8.7343    magn:    0.7007    constr:    0.0000

     total cpu time spent up to now is        5.3 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k =-0.1667 0.1667 0.1667 (   320 PWs)   bands (ev):

     6.3490  11.4672  11.8642  11.8642  12.8202  12.8202  33.0728  38.8766
    40.6548

          k = 0.5000-0.5000 0.8333 (   314 PWs)   bands (ev):

     9.7704  11.4814  11.5692  12.1236  13.0463  15.0483  23.0924  30.5795
    37.7048

          k = 0.1667-0.1667 0.5000 (   313 PWs)   bands (ev):

     8.6807  10.9292  11.8271  12.2749  13.2091  13.3157  28.0617  34.5046
    35.0324

          k =-0.1667-1.1667 0.1667 (   318 PWs)   bands (ev):

     9.9106  10.2895  12.4193  13.0513  13.5727  16.7405  24.9778  26.2089
    30.7833

          k =-0.5000-0.8333-0.1667 (   312 PWs)   bands (ev):

    10.6376  10.7256  11.3359  12.6567  13.4148  19.2199  20.7966  27.8469
    28.5175

          k = 0.5000-0.5000-0.5000 (   332 PWs)   bands (ev):

     9.3493  11.7144  11.7144  13.5012  13.5012  14.0869  19.8453  36.6140
    39.0906

 ------ SPIN DOWN ----------


          k =-0.1667 0.1667 0.1667 (   320 PWs)   bands (ev):

     6.2444  12.2714  12.7102  12.7102  13.5303  13.5303  33.0858  38.6635
    40.6522

          k = 0.5000-0.5000 0.8333 (   314 PWs)   bands (ev):

    10.0902  12.0073  12.2197  12.8390  13.9130  15.6368  23.2977  30.6060
    37.7255

          k = 0.1667-0.1667 0.5000 (   313 PWs)   bands (ev):

     8.6623  11.6491  12.5200  13.0812  13.9657  14.1123  28.1500  34.5205
    34.9422

          k =-0.1667-1.1667 0.1667 (   318 PWs)   bands (ev):

    10.3018  10.9570  13.1007  13.8194  14.4749  17.0243  25.0524  26.3219
    30.7515

          k =-0.5000-0.8333-0.1667 (   312 PWs)   bands (ev):

    11.1706  11.2458  12.0198  13.3486  14.3516  19.4150  21.0518  27.8969
    28.5815

          k = 0.5000-0.5000-0.5000 (   332 PWs)   bands (ev):

     9.8166  12.4307  12.4307  13.9300  14.3737  14.3737  20.2831  36.3686
    39.1148

     the Fermi energy is    14.1193 ev

!    total energy              =     -85.89509969 Ry
     Harris-Foulkes estimate   =     -85.89509969 Ry
     estimated scf accuracy    <          1.3E-13 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =      -2.04110280 Ry
     hartree contribution      =      15.20056975 Ry
     xc contribution           =     -30.10931278 Ry
     ewald contribution        =     -68.94529435 Ry
     smearing contrib. (-TS)   =       0.00004049 Ry

     total magnetization       =     0.64 Bohr mag/cell
     absolute magnetization    =     0.79 Bohr mag/cell

     convergence has been achieved in  13 iterations

     Writing output data file Ni.save

     init_run     :      0.72s CPU      0.83s WALL (       1 calls)
     electrons    :      4.16s CPU      4.31s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.04s CPU      0.04s WALL (       1 calls)
     potinit      :      0.12s CPU      0.12s WALL (       1 calls)

     Called by electrons:
     c_bands      :      1.10s CPU      1.17s WALL (      13 calls)
     sum_band     :      1.25s CPU      1.28s WALL (      13 calls)
     v_of_rho     :      1.27s CPU      1.28s WALL (      14 calls)
     newd         :      0.55s CPU      0.58s WALL (      14 calls)
     mix_rho      :      0.06s CPU      0.08s WALL (      13 calls)

     Called by c_bands:
     init_us_2    :      0.03s CPU      0.03s WALL (     324 calls)
     cegterg      :      1.06s CPU      1.11s WALL (     156 calls)

     Called by sum_band:
     sum_band:bec :      0.00s CPU      0.00s WALL (     156 calls)
     addusdens    :      0.89s CPU      0.90s WALL (      13 calls)

     Called by *egterg:
     h_psi        :      0.83s CPU      0.88s WALL (     418 calls)
     s_psi        :      0.03s CPU      0.03s WALL (     418 calls)
     g_psi        :      0.00s CPU      0.00s WALL (     250 calls)
     cdiaghg      :      0.12s CPU      0.12s WALL (     406 calls)

     Called by h_psi:
     h_psi:pot    :      0.82s CPU      0.88s WALL (     418 calls)
     h_psi:calbec :      0.01s CPU      0.03s WALL (     418 calls)
     vloc_psi     :      0.78s CPU      0.81s WALL (     418 calls)
     add_vuspsi   :      0.04s CPU      0.03s WALL (     418 calls)

     General routines
     calbec       :      0.02s CPU      0.05s WALL (     574 calls)
     fft          :      0.52s CPU      0.56s WALL (     424 calls)
     ffts         :      0.00s CPU      0.01s WALL (      54 calls)
     fftw         :      0.84s CPU      0.84s WALL (    7722 calls)
     interpolate  :      0.08s CPU      0.09s WALL (      54 calls)

     Parallel routines
     fft_scatter  :      0.21s CPU      0.24s WALL (    8200 calls)

     PWSCF        :     5.21s CPU         5.51s WALL


   This run was terminated on:  11:22:21  17Jul2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
